libs/community/tests/integration_tests/vectorstores/test_bigquery_vector_search.py

```diff
@@ -0,0 +1,88 @@
+"""Test BigQuery Vector Search.
+In order to run this test, you need to install Google Cloud BigQuery SDK
+pip install google-cloud-bigquery
+Your end-user credentials would be used to make the calls (make sure you've run
+`gcloud auth login` first).
+"""
+
+import uuid
+
+import pytest
+from google.cloud import bigquery
+
+from langchain_community.vectorstores.bigquery_vector_search import BigQueryVectorSearch
+
+
+@pytest.fixture(scope="class")
+def store(request: pytest.FixtureRequest) -> BigQueryVectorSearch:
+    """BigQueryVectorStore tests context."""
+    TestBigQueryVectorStore.store = BigQueryVectorSearch(
```

Comment:
`embedding` and `table_name` are mandatory parameters.
Also, dataset is not being created automatically by `BigQueryVectorSearch`, so it must be created here.